src/features/game/events/landExpansion/fertiliseCrop.ts

```diff
@@ -20,29 +18,14 @@ type Options = {
 
 export enum FERTILISE_CROP_ERRORS {
   EMPTY_PLOT = "Plot does not exist!",
-  EMPTY_CROP = "There is no crop planted!",
+  CROP_EXISTS = "There is a crop planted!",
   READY_TO_HARVEST = "Crop is ready to harvest!",
   CROP_ALREADY_FERTILISED = "Crop is already fertilised!",
   NO_FERTILISER_SELECTED = "No fertiliser selected!",
   NOT_A_FERTILISER = "Not a fertiliser!",
   NOT_ENOUGH_FERTILISER = "Not enough fertiliser!",
 }
 
-const getPlantedAt = (
-  fertiliser: CropCompostName,
-  plantedAt: number,
-  fertilisedAt: number,
-  cropDetails: Crop
-) => {
-  const timeToHarvest = cropDetails.harvestSeconds * 1000;
-  const harvestTime = plantedAt + timeToHarvest;
-  const timeReduction = (harvestTime - fertilisedAt) / 2;
-  if (fertiliser === "Rapid Root") {
-    return plantedAt - timeReduction;
-  }
-  return plantedAt;
-};
-
 export function fertiliseCrop({
```

Comment:
With the new model (fertilize before planting), this should probably be renamed to `fertilisePlot` since there is no crop (or crop seed even chosen) at the time the method is invoked.